app/models/topic.rb

```diff
@@ -1093,7 +1093,14 @@ def participants_summary(options = {})
     @participants_summary ||= TopicParticipantsSummary.new(self, options).summary
   end
 
-  def make_banner!(user)
+  def make_banner!(user, bannered_until = "")
+    bannered_until ||= ''
+
+    bannered_until = begin
```

Comment:
I really don't get this, why are we suppressing parse errors? I see that we do the same for pinned_until so I _guess_ it's okay, but we really should be validating the date provided by the user and show a proper error to them instead of silently nilling the date

Reply:
same behavior than remove pin job